### DISCUSSION TITLE
Text box not clickable when editing workspace name.

### DISCUSSION THREAD

**Discussion by shompoe** [standardnotes/forum#2584] [General]
**Describe the bug**
Text box not clickable when editing workspace name.

**To Reproduce**
1. Go to Switch Workspace menu and click the pen icon.
2. Click somewhere in the text box.
3. The app cancels editing workspace name and switches to that workspace instead.

**Expected behavior**
The cursor should more to the place where you click so that you don't have to delete everything before you type the workspace name.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 20.0.2

**Comment by tclemans** [comment]
Same issue on iOS mobile app.
Selecting the pen icon to edit the workspace name selects the workspace instead of editing it.
